lib/ui/onboarding/onboarding_page_1_view.dart

```diff
@@ -31,33 +31,43 @@ class OnboardingPage1View extends StatelessWidget {
         "icon": "assets/images/Onboarding Yourself.png",
         "title": "Onboarding Yourself",
         "onTap": () async {
-          goToUrl("https://mosip.stoplight.io/");
+          goToUrl("https://docs.mosip.io/1.2.0/modules/registration-client/operator-onboarding");
         },
       },
       {
-        "icon": "assets/images/Synchronising Data.png",
-        "title": "Synchronising Data",
-        "onTap": () {},
+        "icon": "assets/images/Onboarding Yourself.png",
+        "title": "Registering an Individual",
+        "onTap": () async {
+          goToUrl("https://docs.mosip.io/1.2.0/modules/registration-client/registration-client-home-page");
```

Comment:
instead of hardcoding can we take the URLs from global_param table, check this confgurations
https://github.com/mosip/mosip-config/blob/develop2-v3/registration-default.properties#L376-L381

Reply:
Done.